OpenRobertaServer/staticResources/js/app/roberta/models/progList.model.js

```diff
@@ -27,9 +27,9 @@ define([ 'exports', 'comm' ], function(exports, COMM) {
     /**
      * Refresh example list
      */
-    function loadGalleryList(successFn) {
-        COMM.json("/program/gallery", {
-        }, successFn, "load gallery list");
+    function loadGalleryList(successFn, filters) {
+        var data = filters ?? {};
+        COMM.json("/program/gallery", data, successFn, "load gallery list");
```

Comment:
The REST call doesn't return the filtered objects anymore, always all robots are returned. Please check the value of data.